chunky/src/res/se/llbit/chunky/ui/render/HelpTab.fxml

```diff
@@ -12,7 +12,7 @@
       <padding>
          <Insets bottom="10.0" left="10.0" right="10.0" top="10.0"/>
       </padding>
-      <Label text="Camera key bindings:">
+      <Label text="Camera key bindings (standard projection):">
```

Comment:
All projections but isometric projection, right? :thinking:

Reply:
That would be correct.

`(Excluding isometric)`?

Reply:
Moving the camera up/down (and actually also the other directions) actually still work in isometric projection and can be important to stop the camera from clipping into the ground

Reply:
Confirmed. The isometic camera still has the same controls and they work basically the same.. just a bit differently,